egs/multi_en/s5/README.md

```diff
@@ -2,13 +2,13 @@ This is a WIP **English LVCSR recipe** that trains on data from multiple corpora
 * Fisher (1761 hours)
 * Switchboard (317 hours)
 * WSJ (81 hours)
-* AMI (78 hours IHM, 77 hours SDM)
+* HUB4 (76 hours)
```

Comment:
Probably better say "HUB4-EN" or something, as the HUB4 itself was in a couple different language. Or use the full name: "HUB4 Broadcast News"

Reply:
addressed